packages/terra-status-view/src/StatusView.jsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Divider from 'terra-divider';
+import styles from './StatusView.scss';
+
+const cx = classNames.bind(styles);
+
+const variants = {
+  NODATA: 'no-data',
+  NOMATCHINGRESULTS: 'no-matching-results',
+  NOTAUTHORIZED: 'not-authorized',
+  ERROR: 'error',
+  CUSTOM: 'custom',
+};
+
+const propTypes = {
+  /**
+   * An array of actionable buttons to display.
+   */
+  children: PropTypes.node,
+
+  /**
+   *  Determines if the content should be aligned vertically at the top of the container rather than in the middle.
+   */
+  isAlignedTop: PropTypes.bool,
+
+  /**
+   * Determines if the glyph should be displayed.
+   */
+  isGlyphHidden: PropTypes.bool,
+
+  /**
+   * The descriptive text to display.
+   */
+  message: PropTypes.string,
+
+  /**
+   * The required status title to display. Status views with variants of type no-data, no-matching-results,
+   * not-authorized, and error will have defaulted values unless overridden with this prop.
+   */
+  title: PropTypes.string,
```

Comment:
The description says title is required. Should probably change to `title: PropTypes.string.isRequired,`

Reply:
title is defaulted in some cases, updated the documentation to leave out it being required